### ISSUE TITLE
When I run the command "trade.py import -P eddblink -O all", I get error messages !

### ISSUE THREAD

**Issue by nepomuk16321** [eyeonus/Trade-Dangerous#142]
Hello, as requested, I have opened this issue.
You can read about the start of my problems here: https://forums.frontier.co.uk/threads/trade-dangerous-est-2015-power-users-highly-configurable-trade-optimizer.441509/post-10361659
...
The last action to find the problem was to run "locale_test.zip". The files were requested and downloaded.
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/34adbf12-bd4d-4982-80ba-bbc6069defd5)
Ok, what can I do next ?

**Comment by Tromador** [comment]
I'm not having the same issue as you and I'm running on windows with Python 3.12 also. So I am wondering if there's some bit of badly installed Python modules hiding out in an unexpected place. Python installer seems to have a thing about putting stuff into users profiles if you so much as look at it as a non-admin user. 

So clean it. Check your profile and default profile for any Python site files, or other libraries or modules it's squirrelled away in there. I realise this is a pain in the butt, however given it works fine for me one same OS and Python version, I'm inclined to believe it's Just You™ (sorry).

Once you are convinced there's no more old Python files lurking, reinstall Python using the windows installer and the appropriate options to make it an install for all users. (Which makes it install into Program Files like any other sensible piece of software). 

Then run a cmd windows *as administrator* and use pip to install tradedangerous (and dependant modules) from there. By doing that it also installs those modules as system files, in the Python folder it made at install time and not in user profiles or anywhere else. The command 'pip' should not be used outside of an admin window.

You can then run TD as a non-privileged user just fine, but remember any other modules, or upgrades to TD must be installed as admin to keep everything organised and well, frankly, sane.

**Comment by eyeonus** [comment]
Thank you.

Your problem is indeed caused by having a non-English locale as default. (I couldn't test myself because I have en_US as my default locale.)

Now that I know that, I can implement the fix.

I've pushed a change to the test branch, would you please download it (using the same link I gave you already) and run `python .\trade.py import -P eddblink -www`?

You should get a message like the following: 
```
#Command line was: ['./trade.py', 'import', '-P', 'eddblink', '-www']
#Using "en_US.UTF-8" locale for parsing modified timestamps. Please include this information in any error reports.
NOTE: Checking for update to 'System.csv'.
```

If you do successfully get `Using ... locale ...`, let me know it was a success. (NOTE: The exact locale may not match because the plugin finds the first locale installed on your machine that is English and UTF-8 and uses that.)

@kfsone You have a Mac, can you test this too?

**Comment by nepomuk16321** [comment]
So, downloaded the second test file, unpacked it and executed the above command. Result:
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/3f0e61ce-b605-431b-baf0-a62b8e190991)
The error message is slightly different from the one you expected. I hope this helps you anyway.
And by the way, thank you for taking the trouble to customize TD for me (hopefully not only).
Thank you very much !

**Comment by eyeonus** [comment]
Would you run this and give me the output, please?

[locales.zip](https://github.com/eyeonus/Trade-Dangerous/files/15144151/locales.zip)

**Comment by nepomuk16321** [comment]
... of course ...
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/2346ff32-287e-43a2-82c6-f7753b25ccfd)
... and maybe this screenshot from Windows will also help
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/8540c397-29f3-4079-af85-47cdd0aca8a1)

**Comment by eyeonus** [comment]
I thought as much. Okay, one more time, download the new version, run `python .\trade.py import -P eddblink -www`, let me know what happens. Hopefully I've got it all cleared up now and we can push this to the main branch.

**Comment by kfsone** [comment]
I'll not be able to test on the mac (which botched an update and is currently doing a recovery) until tomorrow.

**Comment by nepomuk16321** [comment]
Ok, I downloaded this version again from the link in the forum. I hope that was correct.
And I have just executed the command. I can't take a screenshot because there is a lot of data running across the screen.
Let's see what the result is.
Ok, a screenshot was possible after all (the data continues to run through):
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/df6a5a48-7c96-43f2-910e-babb6cb18447)
...
